.github/workflows/ios_build_and_test.yml

```diff
@@ -2,22 +2,18 @@ name: ios_build_and_test
 
 on:
   push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
 
 jobs:
   build:
     strategy:
       matrix:
         macos: [macos-latest]
-        xcode: ['12.4']
+        xcode: ['14.2']
     runs-on: ${{ matrix.macos }}
     steps:
     - uses: actions/checkout@v2
-    - uses: sinoru/actions-setup-xcode@v1.1.2
-      with:
-        xcode-version: ${{ matrix.xcode }}
+    - name: Select Xcode version
+      run: sudo xcode-select -s '/Applications/Xcode_${{ matrix.xcode }}.app/Contents/Developer'
```

Comment:
Just FYI, `macos-latest` now includes Xcode 14.3:

https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md

Reply:
Ah, great.  I'll update this.